pkg/webui/components/search-button/index.js

```diff
@@ -0,0 +1,67 @@
+// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React, { useCallback, useContext } from 'react'
+import classnames from 'classnames'
+
+import Button from '@ttn-lw/components/button-v2'
+import Icon from '@ttn-lw/components/icon'
+
+import SidebarContext from '@ttn-lw/containers/side-bar/context'
```

Comment:
Also, this component is exclusive to the sidebar, so it should just be incorporated into the sidebar container rather than being defined as global component.